### ISSUE TITLE
Can't gem install version 0.9.1

### ISSUE THREAD

**Issue by JacobEvelyn** [opal/opal#1282]
I discovered today that I can't install Opal v0.9.1 via either Bundler or `gem install`. I get this error:

```
Gem::Package::PathError: installing into parent path /Users/jacobevelyn/.gem/ruby/2.3.0/gems/js-yaml/bin/js-yaml.js of /Users/jacobevelyn/.gem/ruby/2.3.0/gems/opal-0.9.1 is not allowed
```

I can install 0.9.0 fine.

**Comment by elia** [comment]
I know what that is, Ruby 2.3.0, I'll issue 0.9.2 in a few moments building it from 2.2.3

**Comment by elia** [comment]
@JacobEvelyn ok, please try 0.9.2

**Comment by JacobEvelyn** [comment]
0.9.2 works, thanks!

**Comment by svoboda-jan** [comment]
Getting the same error with v. `0.9.3`, ruby `2.3.1p112`.

**Comment by elia** [comment]
Can you tell me your rubygems version?

Anyway should work if you upgrade rubygems, at the same time I'll try to find the right version from which is safe to build.

**Comment by svoboda-jan** [comment]
Rubygems v. `2.5.1`. After update to `2.6.3` it worked.

**Comment by elia** [comment]
Can anyone with rubygems 2.5.1 (or any other verstion that has problems with opal-0.9.3) try this http://cl.ly/100a0Q26330a/opal-0.9.4.rc1.gem to ensure it works?

**Comment by elia** [comment]
Released 0.9.4, built with Ruby 2.2.3 / Rubygems 2.4.8 after removing all the symlinks from the embedded `node_modules` dir. Hope it's enough 😄
